README.md

```diff
@@ -12,12 +12,20 @@ official artifacts for many historial and modern `solc` versions for Linux and m
 
 The versioned binaries are stored in `~/.solc-select/artifacts/`.
 
-## Quickstart
+## Installation
+
+### Using pip
 
 ```bash
 pip3 install solc-select
 ```
 
+### Using uv (recommended for development)
+
+```bash
+uv pip install solc-select
```

Comment:
`uv tool` might be a better fit for end-user installations (or `uvx` for more ephemeral runs) -- https://docs.astral.sh/uv/concepts/tools/#the-uv-tool-interface

```suggestion
uv tool install solc-select
```

Reply:
Excellent suggestion! Updated to use `uv tool install solc-select` which is indeed better for end-user installations. Thanks for the link to the documentation.